Backend/WebApi-PointMap/Controllers/PointController.cs

```diff
@@ -43,13 +44,13 @@ public IHttpActionResult Get(string guid)
             }
             catch(Exception e)
             {
-                return ResponseMessage(LocalErrorHandler.HandleDatabaseException(e, _db));
+                return ResponseMessage(PointErrorHandler.HandleException(e, _db));

```

Comment:
First exception handler is always SomeErrorHandler for SomeController, which passes the exception to the DatabaseErrorHandler, which passes to the HttpErrorHandler, which passes to the AuthorizationErrorHandler, which passes to the GeneralErrorHandler.

SomeErrorHandler -> DatabaseErrorHandler -> HttpErrorHandler -> AuthorizationErrorHandler -> GeneralErrorHandler.